src/sdk.js

```diff
@@ -279,3 +306,25 @@ Smartcar.AccessDenied = class extends Error {
     this.name = 'AccessDenied';
   }
 };
+
+/**
+ * Vehicle incompatible error returned by authorization flow.
+ *
+ * @extends Error
+ */
+Smartcar.VehicleIncompatible = class extends Error {
+  /**
+   * @param {String} message - detailed error description
+   * @param {Object} vehicleInfo - If a vehicle is incompatible, the user has
```

Comment:
Fixed. Side-note: I'm not sure if `{?param}` is correct JSDoc syntax ([source](https://stackoverflow.com/a/29441515/9045128) - original source is down). Looks like `{param} [name]` is one way to indicate this that I've seen elsewhere in the JS ecosystem, so I went with that.

Reply:
So there's actually a distinction between the two `?` indicates a nullable field ([source](https://jsdoc.app/tags-type.html)) where as `[]` indicate a field which is optional.

In this case it should be `?` because that object is either null or the object